python/tvm/topi/arm_cpu/conv2d_int8.py

```diff
@@ -39,10 +40,10 @@ def _get_default_config(cfg, data, kernel, strides, padding, dilation, out_dtype
     wkl = _get_conv2d_workload(data, kernel, strides, padding, dilation, out_dtype)
     is_kernel_1x1 = wkl.kernel_h == 1 and wkl.kernel_w == 1
     if is_kernel_1x1:
-        conv2d_generic.fallback_schedule_cpu_1x1_int8(cfg, wkl, int32_lanes=2, num_int8_elements=4)
+        conv2d_generic.fallback_schedule_cpu_1x1_int8(cfg, wkl, int32_lanes=4, num_int8_elements=4)
```

Comment:
What's the reasoning behind this change in int32_lanes? Did we use the wrong value all along?

Reply:
Yes, it was the wrong value all along. If you look down at `schedule_conv2d_NCHWc_int8` (https://github.com/apache/tvm/blob/2c0a7c2a7e1df8203c14e4f0326a99aeaceb967a/python/tvm/topi/arm_cpu/conv2d_int8.py#L110-L116), `int32_lanes` was set to 4 already.

Reply:
Good catch